### PR TITLE
feat(prometheus): added dashboards for prometheus-remote-write

### PR DESCRIPTION
**Pull request by tclayton-newr**
# Summary

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] ~Did you check that your alerts actually work?~ N/A
- [x] ~Did you include an InstallPlan and Documentation reference?~ N/A
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.
image
image

### CODE REVIEW

**Comment by jpvajda** [comment]
@tclayton-newr Thanks for the contribution, could you sign the CLA? You just need to do this once to contribute to our New Relic OSS repositories.

File: quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json [hunk 1]
```diff
@@ -0,0 +1,782 @@
...
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT rate(datapointcount(), 1 second) AS 'Datapoints Received / Second' WHERE instrumentation.name = 'remote-write' and instrumentation.provider = 'prometheus' FACET instrumentation.source LIMIT MAX RAW "
+              }
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDY2MDg"
```
Comment (jpvajda) [inline quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:45]:
```suggestion
```

Comment (jpvajda) [reply quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:45]: Please change this value to null

File: quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json [hunk 2]
```diff
@@ -0,0 +1,782 @@
...
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT cardinality(metricName) WHERE instrumentation.name = 'remote-write' and instrumentation.provider = 'prometheus' FACET metricName LIMIT MAX"
+              }
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTA5OTcyMTl8VklafERBU0hCT0FSRHwyMDU5NTk"
```
Comment (jpvajda) [inline quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:348]:
```suggestion
```

Comment (jpvajda) [reply quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:348]: please change this value to null

File: quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json [hunk 3]
```diff
@@ -0,0 +1,782 @@
...
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM MetricRaw SELECT rate(sum(prometheus_tsdb_head_samples_appended_total), 1 second) AS 'Samples Collected per Second' WHERE instrumentation.name = 'remote-write' and instrumentation.provider = 'prometheus' FACET instrumentation.source,instance"
+              }
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDY2MDk"
```
Comment (jpvajda) [inline quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:439]:
```suggestion
```

Comment (jpvajda) [reply quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:439]: please change this value to null

File: quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json [hunk 4]
```diff
@@ -0,0 +1,695 @@
...
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT rate(datapointcount(), 1 second) AS 'Datapoints Received / Second' WHERE instrumentation.name = 'remote-write' FACET instrumentation.source RAW LIMIT MAX"
+              }
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDcxNDI"
```
Comment (jpvajda) [inline quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json:45]:
```suggestion
```

Comment (jpvajda) [reply quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json:45]: please change this value to null

File: quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json [hunk 5]
```diff
@@ -0,0 +1,695 @@
...
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT rate(sum(prometheus_tsdb_head_samples_appended_total), 1 second) AS 'Samples Collected per Second' WHERE instrumentation.name = 'remote-write' RAW FACET instrumentation.source,instance LIMIT MAX"
+              }
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDcxNDM"
```
Comment (jpvajda) [inline quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json:352]:
```suggestion
```

Comment (jpvajda) [reply quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json:352]: please change this value to null

**Review by jpvajda — CHANGES_REQUESTED** [review]
Thanks for the contribution, I made some comments on LinkedEntityGuids that should be null.  You can use our sanatize.js script to clean the JSON up really quickly.

https://github.com/newrelic/newrelic-quickstarts#quickstart-components

**Comment by tclayton-newr** [comment]
@jpvajda thank you for the feedback! I've updated the branch a bit and addressed any comments. 
I also ran into a small issue with the sanitizer that wasn't removing the `linkedEntityGuids` as I was expecting it should. I made a quick PR to address future dashboard.json like mine (there were some line returns). 
https://github.com/newrelic/newrelic-quickstarts/pull/861

**Pull request merged**